README.md

```diff
@@ -179,6 +179,88 @@ cpu: Apple M1 Pro
 BenchmarkUnwrap_D_32_WithSetup-10    	  301116	      3619 ns/op	    8608 B/op	      64 allocs/op
 ```
 
+### Command Line Usage
+
+tr31 is a tool for managing both TDES and AES-derived unique keys per transaction (TR-31) key management.
```

Comment:
The tool gets installed as [`tr31cli`](https://github.com/moov-io/tr31/tree/master/cmd/tr31cli) by default because of how the folder is named. We should rename the folder to `tr31`. 

Define what TDES is.

Reply:
Rename folder to tr31 and change TDES to 3Des